docs/podman-create.1.md

```diff
@@ -204,6 +204,9 @@ configuration passed to the container. Typically this is necessary when the
 host DNS configuration is invalid for the container (e.g., 127.0.0.1). When this
 is the case the **--dns** flags is necessary for every run.
 
+The special sigil **none** can be specified to disable creation of **/etc/resolv.conf** in the container by Podman.
```

Comment:
sigil?

Reply:
Seems to fit

sigil - noun - an inscribed or painted symbol considered to have magical power.

Reply:
Not for us Mathematicians.  A little deep in the Thesaurus for me.
The special name, field, constant...